### PR TITLE
Add bosko project handler for e2e tests.

### PR DESCRIPTION
**Pull request by kl52752**

### CODE REVIEW

**Comment by aojea** [comment]
you should use the trick of of a tools.go file to not bring the dependencies to the project and only use them for testing

https://github.com/kubernetes/kubernetes/commit/f981628ce897a96a8a08a2a21c4d6bdb41601fbd
https://github.com/kubernetes/ingress-gce/tree/master/hack/tools

File: e2e/project_holder.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+	"k8s.io/klog/v2"
+	boskosclient "sigs.k8s.io/boskos/client"
+	"sigs.k8s.io/boskos/common"
```
Comment (aojea) [inline e2e/project_holder.go:26]: I see, so this is the file that is pulling all the dependencies, do you have to embed boskos client here

Are there any other options?

@justinsb @bentheelder do you know if we can avoid pulling all these dependencies?

**Comment by bowei** [comment]
Can we fix the dependencies issue? This is pulling in a huge number of files.

File: e2e/project_holder.go (hunk 1 above)
Comment (aojea) [reply e2e/project_holder.go:26]: >  Can we fix the dependencies issue? This is pulling in a huge number of files.

@bowei this is the thread about it :)

**Comment by bowei** [comment]
Ping on this change...

**Comment by bowei** [comment]
Ping.

**Comment by kl52752** [comment]
Bosko project is not needed anymore for setting up the e2e test grid. Closing this PR

**Pull request closed without merging**